src/plugins/ext/debate/Debate.ts

```diff
@@ -85,13 +85,21 @@ export class Debate {
       await this.avatarClientA.connect();
       await this.avatarClientB.connect();
 
-      this.avatarClientA.onAvatarSpeakingChange = (isSpeaking) => {
-        this.onAvatarSpeakingChange && this.onAvatarSpeakingChange(isSpeaking);
-      };
-
-      this.avatarClientB.onAvatarSpeakingChange = (isSpeaking) => {
-        this.onAvatarSpeakingChange && this.onAvatarSpeakingChange(isSpeaking);
-      };
+      this.avatarClientA.addEventListener(
+        'avatarSpeakingChange',
+        (isSpeaking) => {
+          this.onAvatarSpeakingChange &&
+            this.onAvatarSpeakingChange(isSpeaking);
+        },
+      );
+
+      this.avatarClientB.addEventListener(
+        'avatarSpeakingChange',
+        (isSpeaking) => {
+          this.onAvatarSpeakingChange &&
+            this.onAvatarSpeakingChange(isSpeaking);
+        },
+      );
```

Comment:
question: should we remove the listeners in `disconnectAvatars`?

Reply:
Fixed here: https://github.com/agencyenterprise/alpha-ai-avatar-sdk-js/pull/14/commits/40e028fb1a364d7d1c1e9c8a8d061723620d229e